src/natives/shared/win/SharedFunctions.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
```

Comment:
I suppose it should be `Utils.js`. Now the `SharedFunctions.cs` file 'exports' `ForegroundWindowGuard` class, but in fact `ForegroundWindowGuard` is not a `shared function`.
I think it will be ok if you call it `Utils.js` and publish some funcitons (CompareClassName, etc) and the `ForegroundWindowGuard` class